gcs.tf

```diff
@@ -1,3 +1,8 @@
+locals {
+  default_startup_script  = var.vm_image_image_family == local.r_image_family ? templatefile("${path.module}/assets/r-notebook.sh", var.git_config) : templatefile("${path.module}/assets/py-notebook.sh", var.git_config)
+  custom_startup_script   = var.startup_script != "default.sh" ? templatefile("${path.module}/assets/${var.startup_script}", var.git_config) : ""
```

Comment:
Should this be `custom_startup_script   = var.startup_script != "default.sh" ? templatefile("${path.module}/assets/${var.startup_script}", var.git_config) : ""` to `custom_startup_script   = var.startup_script != "default.sh" ? templatefile(var.startup_script, var.git_config) : ""`?

Otherwise you would have to put your custom script in the `assets` directory within the module.

Reply:
Yes the approach of storing the scripts in the calling project makes much more sense. Done and tested!